app/assets/javascripts/dashboard/protocols.js.coffee

```diff
@@ -159,14 +146,37 @@ $(document).ready ->
             $('#modal_place').html(data.modal)
             $('#modal_place').modal 'show'
             $('.service-requests-table').bootstrapTable()
-            $('.service-requests-table').on 'all.bs.table', ->
-              $(this).find('.selectpicker').selectpicker()
+            reset_service_requests_handlers()
+
+      $(document).on 'change', '.complete-details', ->
```

Comment:
Handler / Code to create the ajax request when you click on a form in the dropdown